build.gradle

```diff
@@ -55,9 +58,24 @@ protobuf {
     }
 }
 
-compileKotlin {
-    kotlinOptions.jvmTarget = "1.8"
+startScripts.enabled = false
+
+task cosmasServer(type: CreateStartScripts) {

```

Comment:
Please add `run` task configuration so that server could be started with `gradle run`. currently it fails:
```
[bard@bardtop3 papeeria-edit-history (tkt-5-grpc-server)]$ gradle run

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':run'.
> No main class specified

```

Reply:
Ok, so `gradle run` should start server, not client?

Reply:
yup, we're mostly interested in server part here

Reply:
Done